### ISSUE TITLE
Make multiarg work with env

### ISSUE THREAD

**Issue by patricklodder** [dogecoin/docker#12]
While reviewing #10, I realized that one of the things that doesn't work with env-as-arg right now is multi-arg, i.e. `-addnode=x -addnode=y`. 

We should add something for that.

**Comment by AbcSxyZ** [comment]
Which type of value this type of argument is requiring ? Obviously, we can't use an environment variable twice, so if we want to do something with it, I don't see another way than having a delimiter to separate values.

Or an amazing alternative : `-e ADDNODE1=x -e ADDNODE2=y` :rofl:

**Comment by patricklodder** [comment]
```
$ grep -rn mapMultiArgs.count src/*cpp | sed 's/^.*mapMultiArgs.count..//;s/".*$//' |sort |uniq
```
- `-addnode` => `<ipv4>:<port>` or `<ipv6>:<port>` or `<dnsname>:<port>`
- `-bind` => `<ipv4>:<port>` or `<ipv6>:<port>` 
- `-bip9params` `<int>:<int>:<int>`
- `-connect` => `<ipv4>:<port>` or `<ipv6>:<port>` or `<dnsname>:<port>`
- `-debug` => `<alpha>`
- `-externalip` => `<ipv4>:<port>` or `<ipv6>:<port>` or `<dnsname>:<port>`
- `-loadblock` => `<filename>` (NOTE: will be hard to use anyway)
- `-onlynet` => `<alphanumeric>`
- `-rpcallowip` => `<ipv4>/<int>` or `<ipv6>/<int>`
- `-rpcauth` => `<string>:<string>` (NOTE: this one may be tricky)
- `-rpcbind` => `<ipv4>:<port>` or `<ipv6>:<port>` 
- `-seednode` => `<ipv4>:<port>` or `<ipv6>:<port>` or `<dnsname>:<port>`
- `-uacomment` => `<string>` (!!!)
- `-whitebind` => `<ipv4>:<port>` or `<ipv6>:<port>` 
- `-whitelist` => `<ipv4>/<int>` or `<ipv6>/<int>`

I think `,`

**Comment by xanimo** [comment]
> While reviewing #10, I realized that one of the things that doesn't work with env-as-arg right now is multi-arg, i.e. `-addnode=x -addnode=y`.
> 
> We should add something for that.

i can confirm adding 2 nodes by running `docker run -it <image> --addnode=x --addnode=y` on an image built off main.

**Comment by patricklodder** [comment]
That's correct, but you don't want to do args like that when you k8s. That's what the `-e ADDNODE=x` is for...

**Comment by xanimo** [comment]
ah forgot about k8s. 👍
